components/pipeline-service/production/base/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
-  - git::https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=e9c5d3d54095f2c63990788e2f2e21f37a7540f7
+  - git::https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=12862f674f40f65cf94d48691e3fe1a2d3bf95a6
```

Comment:
corresponds to https://github.com/openshift-pipelines/pipeline-service/commit/12862f674f40f65cf94d48691e3fe1a2d3bf95a6

Reply:
and https://github.com/redhat-appstudio/infra-deployments/blob/287b79808d337afa33b9bd93a04cee46411427cc/components/pipeline-service/staging/base/kustomization.yaml#L11